test/WebJobs.Script.Tests.Analyzers/AssemblyCacheTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
delete this file for now since it's empty?

Reply:
Deleting.